src/IndividualSighting.jsx

```diff
@@ -0,0 +1,44 @@
+import { useNavigate } from "react-router-dom";
+
+const IndividualSighting = ({ year, season, state, county, index }) => {
+  const navigate = useNavigate();
+
+  const handleClick = (e) => {
+    navigate(`/sightings/${index}`);
+  };
+
+  return (
+    <>
+      <>
+        <div
+          onClick={handleClick}
```

Comment:
try to handle the eslint error, dont ignore it, perhaps you can replace the property